crates/bevy_color/src/color_ops.rs

```diff
@@ -61,3 +61,18 @@ pub trait Alpha: Sized {
         self.alpha() >= 1.0
     }
 }
+
+/// Trait with methods for asserting a colorspace is within bounds.
+pub trait ClampColor: Sized {
+    /// Return a new version of this color clamped, with all fields in bounds.
+    fn clamp(&self) -> Self;
+
+    /// Changes all the fields of this color to assert they are within bounds.
+    fn clamp_self(&mut self);
```

Comment:
I ended up adding this method, but this can be strange: Should I remove this?

Reply:
I think the conventional method naming would be `clamped(self)` and `clamp(&mut self)`

Reply:
I was looking at the original f32::clamp, which return Self, but fair, clamped and clamp looks better in this case.